campaigns.json

```diff
@@ -2777,16 +2813,25 @@
     "go": true,
     "lumisize": -1,
     "maxcopies": 1,
-    "SecondaryLocation": [
-       "T1_ES_PIC_Disk",
-       "T1_US_FNAL_Disk"
-    ],
     "SiteBlacklist": [
        "T2_CH_CERN",
        "T2_CH_CERN_HLT"
     ],
-    "secondaries": {
-      "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL16_106X_mcRun2_asymptotic_v16-v1/PREMIX" : {}
+    "secondaries": {    
+      "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL16_106X_mcRun2_asymptotic_v16-v1/PREMIX" :
```

Comment:
Datasets are already at PIC and FNAL. I'm not sure about PIC, but I think it should be fine.

Reply:
This is one we already picked the sites for the none CP5 one we already moved this is weird.